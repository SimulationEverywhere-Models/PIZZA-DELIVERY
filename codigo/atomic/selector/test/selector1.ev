00:00:01:000 Carga_in 1
00:00:02:000 Carga_in 2
00:00:03:000 Carga_in 3
00:00:04:000 Esperar_m1 1
00:00:05:000 Carga_in 4
00:00:06:000 Carga_in 5
00:00:07:000 Esperar_m2 1
00:00:08:000 Carga_in 6
00:00:09:000 Lista_m2 1
00:00:10:000 Carga_in 7
00:00:11:000 Lista_m1 1
