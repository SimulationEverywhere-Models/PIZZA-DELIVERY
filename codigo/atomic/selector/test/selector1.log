Mensaje I / 00:00:00:000 / Root(00) para top(01)
Mensaje I / 00:00:00:000 / top(01) para selector(02)
Mensaje D / 00:00:00:000 / selector(02) / ... para top(01)
Mensaje D / 00:00:00:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:01:000 / Root(00) / carga_in /      1.00000 para top(01)
Mensaje X / 00:00:01:000 / top(01) / carga_in /      1.00000 para selector(02)
Mensaje D / 00:00:01:000 / selector(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:01:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:01:000 / Root(00) para top(01)
Mensaje * / 00:00:01:000 / top(01) para selector(02)
Mensaje Y / 00:00:01:000 / selector(02) / sel_out1 /      1.00000 para top(01)
Mensaje D / 00:00:01:000 / selector(02) / ... para top(01)
Mensaje Y / 00:00:01:000 / top(01) / sel_out1 /      1.00000 para Root(00)
Mensaje D / 00:00:01:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:02:000 / Root(00) / carga_in /      2.00000 para top(01)
Mensaje X / 00:00:02:000 / top(01) / carga_in /      2.00000 para selector(02)
Mensaje D / 00:00:02:000 / selector(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:02:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:02:000 / Root(00) para top(01)
Mensaje * / 00:00:02:000 / top(01) para selector(02)
Mensaje Y / 00:00:02:000 / selector(02) / sel_out1 /      2.00000 para top(01)
Mensaje D / 00:00:02:000 / selector(02) / ... para top(01)
Mensaje Y / 00:00:02:000 / top(01) / sel_out1 /      2.00000 para Root(00)
Mensaje D / 00:00:02:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:03:000 / Root(00) / carga_in /      3.00000 para top(01)
Mensaje X / 00:00:03:000 / top(01) / carga_in /      3.00000 para selector(02)
Mensaje D / 00:00:03:000 / selector(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:03:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:03:000 / Root(00) para top(01)
Mensaje * / 00:00:03:000 / top(01) para selector(02)
Mensaje Y / 00:00:03:000 / selector(02) / sel_out1 /      3.00000 para top(01)
Mensaje D / 00:00:03:000 / selector(02) / ... para top(01)
Mensaje Y / 00:00:03:000 / top(01) / sel_out1 /      3.00000 para Root(00)
Mensaje D / 00:00:03:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:04:000 / Root(00) / esperar_m1 /      1.00000 para top(01)
Mensaje X / 00:00:04:000 / top(01) / esperar_m1 /      1.00000 para selector(02)
Mensaje D / 00:00:04:000 / selector(02) / ... para top(01)
Mensaje D / 00:00:04:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:05:000 / Root(00) / carga_in /      4.00000 para top(01)
Mensaje X / 00:00:05:000 / top(01) / carga_in /      4.00000 para selector(02)
Mensaje D / 00:00:05:000 / selector(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:05:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:05:000 / Root(00) para top(01)
Mensaje * / 00:00:05:000 / top(01) para selector(02)
Mensaje Y / 00:00:05:000 / selector(02) / sel_out2 /      4.00000 para top(01)
Mensaje D / 00:00:05:000 / selector(02) / ... para top(01)
Mensaje Y / 00:00:05:000 / top(01) / sel_out2 /      4.00000 para Root(00)
Mensaje D / 00:00:05:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:06:000 / Root(00) / carga_in /      5.00000 para top(01)
Mensaje X / 00:00:06:000 / top(01) / carga_in /      5.00000 para selector(02)
Mensaje D / 00:00:06:000 / selector(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:06:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:06:000 / Root(00) para top(01)
Mensaje * / 00:00:06:000 / top(01) para selector(02)
Mensaje Y / 00:00:06:000 / selector(02) / sel_out2 /      5.00000 para top(01)
Mensaje D / 00:00:06:000 / selector(02) / ... para top(01)
Mensaje Y / 00:00:06:000 / top(01) / sel_out2 /      5.00000 para Root(00)
Mensaje D / 00:00:06:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:07:000 / Root(00) / esperar_m2 /      1.00000 para top(01)
Mensaje X / 00:00:07:000 / top(01) / esperar_m2 /      1.00000 para selector(02)
Mensaje D / 00:00:07:000 / selector(02) / ... para top(01)
Mensaje D / 00:00:07:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:08:000 / Root(00) / carga_in /      6.00000 para top(01)
Mensaje X / 00:00:08:000 / top(01) / carga_in /      6.00000 para selector(02)
Mensaje D / 00:00:08:000 / selector(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:08:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:08:000 / Root(00) para top(01)
Mensaje * / 00:00:08:000 / top(01) para selector(02)
Mensaje Y / 00:00:08:000 / selector(02) / sel_out3 /      6.00000 para top(01)
Mensaje D / 00:00:08:000 / selector(02) / ... para top(01)
Mensaje Y / 00:00:08:000 / top(01) / sel_out3 /      6.00000 para Root(00)
Mensaje D / 00:00:08:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:09:000 / Root(00) / lista_m2 /      1.00000 para top(01)
Mensaje X / 00:00:09:000 / top(01) / lista_m2 /      1.00000 para selector(02)
Mensaje D / 00:00:09:000 / selector(02) / ... para top(01)
Mensaje D / 00:00:09:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:10:000 / Root(00) / carga_in /      7.00000 para top(01)
Mensaje X / 00:00:10:000 / top(01) / carga_in /      7.00000 para selector(02)
Mensaje D / 00:00:10:000 / selector(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:10:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:10:000 / Root(00) para top(01)
Mensaje * / 00:00:10:000 / top(01) para selector(02)
Mensaje Y / 00:00:10:000 / selector(02) / sel_out3 /      7.00000 para top(01)
Mensaje D / 00:00:10:000 / selector(02) / ... para top(01)
Mensaje Y / 00:00:10:000 / top(01) / sel_out3 /      7.00000 para Root(00)
Mensaje D / 00:00:10:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:11:000 / Root(00) / lista_m1 /      1.00000 para top(01)
Mensaje X / 00:00:11:000 / top(01) / lista_m1 /      1.00000 para selector(02)
Mensaje D / 00:00:11:000 / selector(02) / ... para top(01)
Mensaje D / 00:00:11:000 / top(01) / ... para Root(00)
