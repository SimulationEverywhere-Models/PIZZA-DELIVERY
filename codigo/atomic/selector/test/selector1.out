00:00:01:000 sel_out1 1
00:00:02:000 sel_out1 2
00:00:03:000 sel_out1 3
00:00:05:000 sel_out2 4
00:00:06:000 sel_out2 5
00:00:08:000 sel_out3 6
00:00:10:000 sel_out3 7
